examples/exp_emodb_textclassifier.ini

```diff
@@ -0,0 +1,21 @@
+[EXP]
+root = ./examples/results
+name = emodb_textclassifier
+[DATA]
+databases = ['emodb']
+emodb = ./examples/results//exp_emodb_translate/results/all_predicted.csv
```

Comment:
@felixbur 

How can I get this `all_predicted.csv`? Which example code do I need to run?

I got an error when running this example directly due to the non-existence of the CSV file above.